angular/projects/spark-angular/src/lib/directives/sprk-button/sprk-button.stories.ts

```diff
@@ -15,15 +16,14 @@ export default {
     info: `
 ${markdownDocumentationLinkBuilder('button')}
 ##### When to Use \`<button>\` vs. \`<a>\`
-The Button component can use either a button (\`<button>\`)
-or anchor (\`<a>\`) HTML element. It is very important
-for accessibility to choose the correct element.
+SprkButton can either render as a button (\`<button>\`)
```

Comment:
Since this is a directive, it is sprkButton camel casing and not SprkButton.

Reply:
✅